### PR TITLE
Bug 1791231: Make addresses ipv6 compatible

### PR DESCRIPTION
**Pull request by jcantrill**
Make ES config ipv6 compatible to resolve: 

* [Bug 1791231](https://bugzilla.redhat.com/show_bug.cgi?id=1791231)
* [Bug 1791349](https://bugzilla.redhat.com/show_bug.cgi?id=1791349)

### CODE REVIEW

File: pkg/k8shandler/configuration_tmpl.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ node:
   max_local_storage_nodes: 1
 
 network:
-  host: 0.0.0.0
+  host: _local_
```
Comment (ewolinetz) [inline pkg/k8shandler/configuration_tmpl.go]: seems like these are two different things
`0.0.0.0` means bind on any address
`_local_` to me means only local access is allowed... 
can you confirm?

Comment (jcantrill) [reply pkg/k8shandler/configuration_tmpl.go]: > seems like these are two different things
> `0.0.0.0` means bind on any address
> `_local_` to me means only local access is allowed...
> can you confirm?

```
_local_ | Any loopback addresses on the system, for example 127.0.0.1.
```
* https://www.elastic.co/guide/en/elasticsearch/reference/6.8/modules-network.html#network-interface-values
*https://www.elastic.co/guide/en/elasticsearch/reference/6.8/network.host.html

Comment (ewolinetz) [reply pkg/k8shandler/configuration_tmpl.go]: yeah, this is not what we want then. that means only access to ES would be from itself

Comment (jcantrill) [reply pkg/k8shandler/configuration_tmpl.go]: reverted.  We'll have to address if it fails ipv6

Comment (ewolinetz) [reply pkg/k8shandler/configuration_tmpl.go]: why not make this `_global_` ?

**Pull request merged**